packages/web/src/new/scenes/Main/Home/Subviews/WalletsCollectionView/common/StaticSectionsCollectionView/StaticSectionsCollectionView.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useMemo } from 'react';
+
+import { styled } from '@linaria/react';
+import { computed } from 'mobx';
+import { observer } from 'mobx-react-lite';
+import { nanoid } from 'nanoid';
+
+import type { ISDListViewModel } from 'new/core/viewmodels/SDListViewModel';
+import { SDFetcherState } from 'new/core/viewmodels/SDViewModel';
+
+import { SDCollectionViewItem } from './models/SDCollectionViewItem';
+
+const Wrapper = styled.div``;
+
+interface Props<T> {
+  viewModel: ISDListViewModel<T>;
+  numberOfLoadingCells?: number;
+  renderPlaceholder: (key: string) => React.ReactNode;
+  renderItem: (item: T, index: number) => React.ReactNode;
+  renderEmpty?: (key: string) => React.ReactNode;
+  customFilter?: (item: T) => boolean;
+}
+
+export const StaticSectionsCollectionView = observer(
+  <T,>({
+    viewModel,
+    numberOfLoadingCells = 2,
+    renderPlaceholder,
+    renderItem,
+    renderEmpty,
+    customFilter,
+  }: Props<T>) => {
+    const items = useMemo(
+      () =>
+        computed(() => {
```

Comment:
Do we need to use `computed` in our view logic? Or can we annotate the respective class properties?

Reply:
I didn't find another way to use `useMemo` and `observable` data without using of `useMemo` and `computed`.
And I think we need to write this logic in component because all data we get from props only